packages/portal-proto/src/components/Table/TotalItem.tsx

```diff
@@ -32,7 +32,8 @@ const TotalItems: React.FC<TotalItemsProps> = ({
 
   return (
     <>
-      Total of <b>{formattedTotal}</b> {itemText}
+      Total of <b data-testid="text-total-item-count">{formattedTotal}</b>{" "}
```

Comment:
Having the test id on this nested bold tag is a little strange to me 🤔. We could move it up to the vertical table component [here](https://github.com/NCI-GDC/gdc-frontend-framework/blob/develop/packages/portal-proto/src/components/Table/TableHeader.tsx#L95) but I'm not sure if you're trying to specifically select the table headers with item counts.

Reply:
I put the ID there to have it directly on an item count we may check. If it doesn't hurt current or future formatting i'd like it there. Let me know